src/executor/evaluate/evaluated.rs

```diff
@@ -342,3 +378,25 @@ fn literal_divide(a: &AstValue, b: &AstValue) -> Result<AstValue> {
         _ => Err(EvaluateError::UnreachableLiteralArithmetic.into()),
     }
 }
+
+fn literal_positive(a: &AstValue) -> Result<AstValue> {
+    match a {
+        AstValue::Number(a) => match a.parse::<i64>() {
+            Ok(a) => Ok(AstValue::Number(a.to_string())),
+            _ => panic!(),
+        },
+        AstValue::Null => Ok(AstValue::Null),
+        _ => Err(EvaluateError::UnreachableLiteralArithmetic.into()),
```

Comment:
In fact.. current `evaluated.rs` file has some problems.
`panic!` uses and branches which are named `Unreachable` but actually reachable.

It looks like you followed existing code patterns well like `fn literal_add` functions.
But you don't need to follow bad codes, :) it's my fault.
https://github.com/gluesql/gluesql/issues/52

Good thing is, not like `literal_add` , these are unary operator function so it only takes a single operand.
So it's easier to remove `panic!()`, let's try out.

* `panic!()` in here can easily reproduced by this test case which uses float type `+2.0`.
```sql
SELECT id FROM Hunter WHERE +2.0 > 1
```
You also need to consider float type uses.
In `src/data/value.rs`, `impl TryFrom<&AstValue> for Value {` TryFrom implementation would be good to see.
```rust
AstValue::Number(v) => v
     .parse::<i64>()
     .map_or_else(|_| v.parse::<f64>().map(Value::F64), |v| Ok(Value::I64(v)))
     .map_err(|_| ValueError::FailedToParseNumber.into()),
```
(I'd like to make it simpler but I didn't know what to do at that time I wrote in this kind of way)

* Reachable but named `(EvaluateError::UnreachableLiteralArithmetic.into()),`
```sql
SELECT id FROM Hunter WHERE +"asdf" = -1
```
This sql query above will show `UnreachableLiteralArithmetic`, though `Unrechable-` prefixed errors are not supposed to see.

Let's remove these two `panic!()` and `Unreachable` error.